src/util/contentful/richTextRenderer.js

```diff
@@ -130,6 +131,11 @@ export function richTextRenderer(content) {
 						alt="${formattedResponsiveImageSet?.description}"
 						:source-sizes="${sourceSetArrayAsString}" />`;
 		}
+		if (isFAQ) {
+			return `<kv-frequently-asked-questions
+						:questions="${htmlSafeStringify(entryContent?.fields?.contents)}"
```

Comment:
Generally, this looks right? Was it operating correctly for you with this `htmlSafeStringify` wrapper?